include/circt/Dialect/Handshake/Simulation.h

```diff
@@ -17,6 +17,11 @@
 
 // Functions used to execute a restricted form of the standard dialect, and
 // the handshake dialect.
+#include <stdio.h>
```

Comment:
I don't think this header is needed here.  It is generally a good idea to avoid c headers in public circt headers where possible, and I'd recommend using `<cstdio>` instead of `<stdio.h>` when you need it.